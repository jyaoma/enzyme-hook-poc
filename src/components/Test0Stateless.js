import React from 'react';

const Test0Stateless = ({ testProp }) => {
  return <div>
    <div testProp={testProp}>Hello World!</div>
  </div>
};

export default Test0Stateless;
