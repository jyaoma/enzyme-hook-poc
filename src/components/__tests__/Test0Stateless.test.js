import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import Test0Stateless from '../Test0Stateless';

describe('Test0Stateless', () => {
  it('should render correctly', () => {
    const rendered = renderer.create(<Test0Stateless />);
    expect(rendered.toJSON()).toMatchSnapshot();
  });

  it('should be able to see props', () => {
    const tree = shallow(<Test0Stateless testProp="oof" />);
    console.warn(tree.props());
  });
});