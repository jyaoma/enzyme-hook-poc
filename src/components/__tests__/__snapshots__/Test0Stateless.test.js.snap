// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test0Stateless should render correctly 1`] = `
<div>
  Hello World!
</div>
`;
